src/annotator/test/integration/hypothesis-injector-test.js

```diff
@@ -64,37 +68,36 @@ describe('HypothesisInjector integration test', () => {
     container.remove();
   });
 
-  describe('#injectClient', () => {
+  describe('injectClient', () => {
     it('configures client', async () => {
       const frame = document.createElement('iframe');
       container.append(frame);
-      const injector = createHypothesisInjector();
 
-      await injector.injectClient(frame);
+      await injectClient(frame, { clientUrl: 'https://hyp.is' });
 
       const configElement = frame.contentDocument.querySelector(
         '.js-hypothesis-config'
       );
       const config = JSON.parse(configElement.textContent);
 
-      assert.match(config.subFrameIdentifier, /[0-9]+/);
+      assert.match(config.subFrameIdentifier, /[a-f0-9]+/);
       assert.notOk(config.assetRoot);
       assert.notOk(config.notebookAppUrl);
       assert.notOk(config.sidebarAppUrl);
     });
 
     it('copies client asset locations from host frame', async () => {
       hostJSONConfig = {
+        clientUrl: 'chrome-extension:///o',
```

Comment:
It looks like this line is incomplete. Can we make it a realistic-looking URL that matches the pattern of the others.